internal/config/config_helpers.go

```diff
@@ -27,3 +34,15 @@ func validateRateLimit(rate float64) bool {
 	}
 	return true
 }
+
+func validatePartialFlushMinSpans(minSpans int) bool {
+	if minSpans < 0 {
+		log.Warn("ignoring DD_TRACE_PARTIAL_FLUSH_MIN_SPANS: negative value %d", minSpans)
+		return false
+	}
+	if minSpans > TraceMaxSize {
```

Comment:
In the deleted code (ddtrace/tracer/option.go L427-430), `minSpans` values == `0` or `TraceMaxSize` were considered invalid. Should we keep that here?